src/cmd/cli/command/commands.go

```diff
@@ -614,14 +615,23 @@ var tailCmd = &cobra.Command{
 		var etag, _ = cmd.Flags().GetString("etag")
 		var raw, _ = cmd.Flags().GetBool("raw")
 		var since, _ = cmd.Flags().GetString("since")
+		var utc, _ = cmd.Flags().GetBool("utc")
 
-		ts, err := cli.ParseTimeOrDuration(since)
+		if utc {
+			os.Setenv("TZ", "UTC")
```

Comment:
I don't see that we are referencing the env "TZ" anywhere.

Reply:
Golang does that